app/lib/test_applications.rb

```diff
@@ -251,6 +254,15 @@ def withdraw_offer(choice)
     choice.audits.last&.update_columns(created_at: time)
   end
 
+  def defer_offer(choice)
+    as_provider_user(choice) do
+      fast_forward(1..3)
+      # TODO: should be calling a service for this - not yet implemented
+      choice.update_columns(status: 'offer_deferred', updated_at: time)
```

Comment:
Maybe we should just implement the service now including `ApplicationStateChange`, however thin it is, then this code doesn't need to change again?

Reply:
I've added a minimal `DeferOffer` service.